src/Symfony/Component/Routing/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+4.4.0
+-----
+
+ * deprecated `ServiceRouterLoader` in favor of `ContainerLoader`
+ * deprecated `ObjectRouteLoader` in favor of `ObjectLoader`
+ * deprecated `ObjectLoader::getServiceObject()` in favor of `ObjectLoader::getObject()`
```

Comment:
new classes must not be introduced with deprecated methods. 
The previous entry is enough: deprecating a class means deprecating all of its API.